eda/tests/unittests/analysis/test_model.py

```diff
@@ -45,6 +45,37 @@ def test_AutoGluonModelEvaluator_regression():
     _assert_importance_is_present(state)
     assert state.model_evaluation.confusion_matrix is None
     assert state.model_evaluation.confusion_matrix_normalized is None
+    assert state.model_evaluation.y_true_val is None
+    assert state.model_evaluation.y_pred_val is None
+
+
+def test_AutoGluonModelEvaluator_regression__with_test_data():
+    df_train = pd.read_csv(os.path.join(RESOURCE_PATH, "houses", "train_data.csv")).sample(100, random_state=0)
+    df_val = pd.read_csv(os.path.join(RESOURCE_PATH, "houses", "test_data.csv"))[:50]
+    df_test = pd.read_csv(os.path.join(RESOURCE_PATH, "houses", "test_data.csv"))[50:101]
```

Comment:
Not an issue, just caught my eye, did you intend `df_test` to have 51 entries while `df_val` has 50 and `df_train` has 100?

Reply:
This is by design to quickly identify what dataset was used in the call.